Dockerfile

```diff
@@ -6,19 +6,18 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
   && buildDeps=' \
-    build-essential \
+    gcc \
     libpq-dev \
-    python-dev \
 	' \
   && deps=' \
     gdal-bin \
-    git \
-    postgresql-client-9.4 \
+    gettext \
+    postgresql-client \
```

Comment:
Pinning to a specific version of postgresql-client seemed like a good idea as postgres releases happen consistently, although not frequently.